test/decorator-request.test.js

```diff
@@ -0,0 +1,24 @@
+import { test } from 'tap'
+import Fastify from 'fastify'
+
+import slowDownPlugin from '../index.js'
+import { DEFAULT_OPTIONS } from '../lib/constants.js'
+
+test('should contain the slowDown request decorator', async t => {
+  const fastify = Fastify()
+  await fastify.register(slowDownPlugin)
+  t.teardown(() => fastify.close())
+
+  fastify.get('/', async req => req.slowDown)
+
+  const response = await fastify.inject('/')
+
+  t.equal(
+    response.body,
+    JSON.stringify({
+      limit: DEFAULT_OPTIONS.delayAfter,
+      current: 1,
+      remaining: DEFAULT_OPTIONS.delayAfter - 1
+    })
```

Comment:
use the response json method and you don't need to stringify the other object. also, we need to test that when there is a delay, we're getting it in the slowDown recorator